app/models/comment.rb

```diff
@@ -5,6 +5,6 @@ class Comment < ApplicationRecord
   after_save :update_counter
 
   def update_counter
-    post.increment!(:commentsCounter)
+    post.increment!(:comments_count)
```

Comment:
- [ ] I think you have misnamed two variables because somewhere it is comments_count because I cant create posts from database
![image](https://user-images.githubusercontent.com/62705947/186730536-52519612-a2eb-42b4-94b4-8b4d04b580cf.png)